frontend/src/components/common/formQuestions/QuestionsAccordionItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import {
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Text,
+} from "@chakra-ui/react";
+
+import { FormQuestion } from "../../../types/CampsTypes";
+import QuestionCard from "./QuestionCard";
+
+type QuestionsAccordionItemProps = {
+  fixedQuestions: Array<FormQuestion>;
+  dynamicQuestions: Array<FormQuestion>;
```

Comment:
I know we avoid comments since everything is already fairly verbose, but can you leave some comments here explaining what "fixed" and "dynamic" means?

Reply:
> The Camper Information and Emergency Contact sections seem fine.
> 
> One blocking item -- is the "Camper Specific Information" questions out of scope?

Admin can't add "camp specific" questions to the form template

Reply:
> > The Camper Information and Emergency Contact sections seem fine.
> > One blocking item -- is the "Camper Specific Information" questions out of scope?
> 
> Admin can't add "camp specific" questions to the form template

talked async, putting this in a separate PR